.github/workflows/transifex.yml

```diff
@@ -0,0 +1,39 @@
+name: Transifex
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron:  '25 12 * * 0'
+
+jobs:
+  build:
+    name: ${{ matrix.type }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        type: [update-templates, update-translations]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Install dependencies
+        run: |
+          sudo apt-get -qq update
+          sudo apt-get install -y gettext translate-toolkit transifex-client
+      - name: Configure Transifex
+        run: |
+          echo "[https://www.transifex.com]\n" > ~/.transifexrc
+          echo "hostname = https://www.transifex.com\n" >> ~/.transifexrc
+          echo "username = api\n" >> ~/.transifexrc
+          echo "token = ${{ secrets.TRANSIFEX_API_KEY }}\n" >> ~/.transifexrc
```

Comment:
Also I believe the `secrets.TRANSIFEX_API_KEY` is empty.

Reply:
No, it's not. I added it, you will just never see it on CI